gsonpath-extensions-compiler/src/main/java/gsonpath/extension/def/stringdef/StringDefGsonPathFieldValidator.java

```diff
@@ -0,0 +1,15 @@
+package gsonpath.extension.def.stringdef;
+
+import com.google.auto.service.AutoService;
+import gsonpath.compiler.GsonPathExtension;
+
+/**
+ * Note: AutoService only works when annotating a Java class.
+ */
+@SuppressWarnings("All")
+@AutoService(GsonPathExtension.class)
+public class StringDefGsonPathFieldValidator extends StringDefGsonPathFieldValidatorImpl {
+    public StringDefGsonPathFieldValidator() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)